### PR TITLE
update pytezos tests

### PR DESCRIPTION
**Pull request by DariusParvin**
update testnet tests to work with pytezos 3.2.4

### CODE REVIEW

**Review by jakinyele — CHANGES_REQUESTED** [review]
Please add a requirements.txt file to reflect the specific version of pytezos.

**Comment by DariusParvin** [comment]
@jakinyele I added a requirements file and updated the readme

File: pytezos-tests/README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Set up python3 virtual environment.
 We'll need to download [PyTezos](https://github.com/baking-bad/pytezos) to interact with the tezos node. 
 ```
     sudo apt-get install libsodium-dev libsecp256k1-dev libgmp-dev
-    pip3 install pytezos
+    pip3 install requirements
```
Comment (jakinyele) [inline pytezos-tests/README.md:22]: btw, i think you meant: `pip3 install -r requirements.txt`

**Review by jakinyele — CHANGES_REQUESTED** [review]
One more thing to fix

File: pytezos-tests/README.md (hunk 1 above)
Comment (DariusParvin) [reply pytezos-tests/README.md]: fixed

**Review by jakinyele — APPROVED** [review]

**Pull request merged**